problem.py

```diff
@@ -69,10 +69,7 @@ def _read_data(path, typ):
     """
     test = os.getenv('RAMP_TEST_MODE', 0)
 
-    if test:
-        suffix = '_mini'
-    else:
-        suffix = ''
```

Comment:
I personally think we should keep this (I know that currently there are not yet new '_mini' data (using the new quadrangles), but I would rather fix that, than letting the `--quick-mode` run on all the data (which will be far from quick).

The structure of the data did not change I think? So keeping the old '_mini' data is maybe also not that much of a problem?

Reply:
^^